python/ray/tests/test_placement_group.py

```diff
@@ -1156,5 +1162,93 @@ def assert_num_cpus(expected_num_cpus):
     wait_for_condition(lambda: assert_num_cpus(num_nodes * num_cpu_per_node))
 
 
+@pytest.mark.parametrize(
+    "ray_start_cluster_head", [
+        generate_system_config_map(
+            num_heartbeats_timeout=20, ping_gcs_rpc_server_max_retries=60)
+    ],
+    indirect=True)
+def test_create_pg_after_gcs_server_restart(ray_start_cluster_head):
```

Comment:
```suggestion
def test_create_placement_group_after_gcs_server_restarts(ray_start_cluster_head):
```

Reply:
same for other places, avoid using abbreviations.

Reply:
fixed